store/mockstore/unistore/cophandler/analyze.go

```diff
@@ -132,20 +156,34 @@ type analyzeIndexProcessor struct {
 	statsBuilder *statistics.SortedBuilder
 	cms          *statistics.CMSketch
 	rowBuf       []byte
+
+	topNCount        int32
+	topNValuePairs   []statistics.TopNMeta
+	topNCurValuePair statistics.TopNMeta
 }
 
-func (p *analyzeIndexProcessor) Process(key, value []byte) error {
+func (p *analyzeIndexProcessor) Process(key, _ []byte) error {
 	values, _, err := tablecodec.CutIndexKeyNew(key, p.colLen)
 	if err != nil {
 		return err
 	}
 	p.rowBuf = p.rowBuf[:0]
+
 	for _, val := range values {
 		p.rowBuf = append(p.rowBuf, val...)
 		if p.cms != nil {
 			p.cms.InsertBytes(p.rowBuf)
 		}
 	}
+	if bytes.Compare(p.topNCurValuePair.Encoded, p.rowBuf) == 0 {
```

Comment:
Please add some comments here to indicate that the scan is orderly so that only comparing it with `rowBuf` is enough.